syllabus/7-Polkadot/README.md

```diff
@@ -2,22 +2,66 @@
 
 ## Learning Outcomes
 
+* Mission
+    * What are the goals behind Polkadot?
+* Elections
+    * How do we get validators to play our game?
+    * NPoS
+* Consenus
+    * How validators play the consensus game?
+    * Data collation
+    * Availability sharding
+    * Execution sharding
+    * Scheduling (Coretime)
+    * XCMP
+* Governance
+    * How does the system evolve?
+    * OpenGov
+    * Fellowship
+* Tooling
+    * What extra tools are available to build on Polkadot?
+    * Cumulus
+    * Zombienet
+
 ## Hands on activities
 
-## Schedule
+- Build a parachain (maybe)
```

Comment:
It is best to movet these practical parts into polkadot-sdk module now. Would that work?